### DISCUSSION TITLE
Supporting many languages, adding additional field to context menu

### DISCUSSION THREAD

**Discussion by RinnetenseiQ** [obiscr/ChatGPT#164] [Ideas]
Is there any possibilty to change language of this options?
![image](https://user-images.githubusercontent.com/49791012/229584971-aca6e49a-621a-422d-8631-1df1e401d94f.png)
Can you provide interface to edit/add own fields to context menu?

**Comment by obiscr** [comment]
In fact, this feature has been supported for a long time, you can refer here: https://chatgpt.en.obiscr.com/settings/custom-prefix-settings/#how-to-use
